server/routes/dashboard/dashboard.ts

```diff
@@ -85,37 +86,12 @@ export default function dashboard(service: Services): Router {
       page,
```

Comment:
there is no validation of `location` with respect to user’s caseloads meaning you can see reports you shouldn't